book/src/generated/typable-cmd.md

```diff
@@ -53,3 +53,5 @@
 | `:sort` | Sort ranges in selection. |
 | `:rsort` | Sort ranges in selection in reverse order. |
 | `:tree-sitter-subtree`, `:ts-subtree` | Display tree sitter subtree under cursor, primarily for debugging queries. |
+| `:refresh-config` | Refreshes helix's config. |
```

Comment:
Let's call it `:reload-config`. Better yet, maybe `:config-reload`, `:config-open` so they're grouped in the list